src/utils/resolveModule.js

```diff
@@ -0,0 +1,16 @@
+import resolve from 'resolve';
+
+export default module => {
```

Comment:
Maybe show that module is a param to the fat arrow function?
```js
export default (module) => {
```

Also, it's not totally clear that the fat arrow is better than `function` when export default is used.


```js
export default function(module) {
```